dynamo-circe/src/main/scala/d4s/codecs/circe/D4SCirceEncoder.scala

```diff
@@ -0,0 +1,32 @@
+package d4s.codecs.circe
+
+import d4s.codecs.D4SEncoder
+import io.circe._
+import io.circe.syntax._
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+
+import scala.jdk.CollectionConverters._
+
+trait D4SCirceEncoder[T] extends D4SEncoder[T] {
+  def encode(item: T): Map[String, AttributeValue]
+}
```

Comment:
`D4SEncoder` already has encode method. I guess we can declare it  like decoder here https://github.com/PlayQ/d4s/pull/7/files#diff-8c3a3286cf49b3a2777b7e243ac29a5aR12